src/templates/ja/2020/methodology.html

```diff
@@ -360,39 +365,85 @@ <h3 id="rework-css"><a href="#rework-css" class="anchor-link">Rework CSS</a></h3
+      </p>
+    </section>
+
+    <section>
+        <h4 id="parsel"><a href="#parsel" class="anchor-link">Parsel</a></h4>
+
+        <p>
+            <a href="https://projects.verou.me/parsel/">Parsel</a>はCSSセレクタパーサとspecificity calculatorで、元々は<a href="{{ url_for('chapter', year=year, lang=lang, chapter='css') }}">CSS</a>の章をリードする<a href="{{ url_for('contributors', year=year, lang=lang, _anchor='LeaVerou') }}">Lea Verou</a>と別のライブラリとしてオープンソース化されています。
+            これは、セレクタと特異性に関連するすべてのCSSメトリクスで広く使われています。
+        </p>
+    </section>
+
     <section>
       <h2 id="process"><a href="#process" class="anchor-link">分析プロセス</a></h2>
 
       <p>
-        Web Almanacの計画と実行には、Webコミュニティの数十人の貢献者の協力を得て、約1年を要しました。このセクションでは、Web Almanacに掲載されているメトリクスを選択した理由、それらがどのようにして照会されたか、そして解釈されたかについて説明します。
+        Web Almanacの計画と実行には、100人以上のWebコミュニティの<a href="{{ url_for('contributors', year=year, lang=lang) }}">投稿者</a>の協力を得て、約1年を要しました。このセクションでは、Web Almanacに掲載されているチャプターを選んだ理由、それらのメトリクスがどのようにして照会されたか、そしてどのように解釈されたかを説明します。
       </p>
     </section>
 
     <section>
-      <h3 id="brainstorming"><a href="#brainstorming" class="anchor-link">ブレインストーミング</a></h3>
+      <h3 id="planning"><a href="#planning" class="anchor-link">プランニング</a></h3>
+
+      <p>
+        2020年のWeb Almanacは、<a href="https://twitter.com/rick_viscomi/status/1273135952848977920">2020年6月</a>にキックオフしましたが、COVID-19に関連した不安や社会正義の抗議活動のため、2019年のタイムラインよりも遅くなりました。2020年のこれらの出来事やその他の出来事は、制作プロセス全体の下火となり、このようなテンポの速いプロジェクトのストレスを超えて、私たちの貢献者に多くの追加的な負担をかけました。
+      </p>
+
+      <p>
+        昨年の方法論で<a href="https://almanac.httparchive.org/en/2019/methodology#brainstorming">述べたように、</a>。
+      </p>
+
+      <blockquote>
+        Web Almanacの今後のエディションの明確な目標の1つは、著者やレビュワーとして、代表性の低い声や異質な声をより多く取り入れることを奨励することです。
+      </blockquote>
 
       <p>
-        2020年Web Almanacは、2020年6月に<a href="https://twitter.com/rick_viscomi/status/1273135952848977920?s=20">新規執筆者を募集して</a>スタートしました。ブレーンストーミングのラウンドの後、22の章が固まり、私たちは<a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues/876">各章に主題専門家とレビュワーを割り当てる作業を開始しました</a>。このプロセスには、この規模のプロジェクトにコミットするためにボランティアを集めるという課題があったため、いくつかの固有の偏りがありました。したがって、寄稿者の多くは同じ専門家サークルのメンバーです。Web Almanacの今後のエディションの明確な目標の1つは、著者やレビュワーとして、代表性の低い、異質な声をより多く取り入れることを奨励することです。
+        そのために、今年は著者の探し方、選び方を体系的に変えてみました。
+
+        <ul>
+          <li>
+            これまでの著者は、異なる視点のためのスペースを確保するために、再び書くことを特別に望まれていませんでした。
+          </li>
+          <li>
+            2020年の著者を支持する人たちは皆、見た目や考え方が似ている人を指名しないように特に意識するように求められました。
+          </li>
+          <li>
+            2019年の執筆者の多くはGoogleの社員であり、今年はより広いウェブコミュニティからの視点をよりバランスよく得ることを試みました。Web Alamanacの声はコミュニティそのものを反映したものであるべきであり、反響室を作らないように特定の企業に偏ったものではないと考えています。
+          </li>
+          <li>
+            プロジェクトのリーダーは、すべての著者の推薦を検討し、新しい視点をもたらし、地域社会の中での未発表グループの声を増幅させる著者の選定に尽力しました。
+          </li>
+        </ul>
+
+        将来的には、このプロセスを繰り返していくことで、Web Almanacがより多様で包括的なプロジェクトとなり、さまざまなバックグラウンドを持つ投稿者が参加できるようにしたいと考えています。
```

Comment:
Ah forgot to say that we had to fix this code in #1500 

```suggestion
        そのために、今年は著者の探し方、選び方を体系的に変えてみました。
      </p>
      
      <ul>
        <li>
          これまでの著者は、異なる視点のためのスペースを確保するために、再び書くことを特別に望まれていませんでした。
        </li>
        <li>
          2020年の著者を支持する人たちは皆、見た目や考え方が似ている人を指名しないように特に意識するように求められました。
        </li>
        <li>
          2019年の執筆者の多くはGoogleの社員であり、今年はより広いウェブコミュニティからの視点をよりバランスよく得ることを試みました。Web Alamanacの声はコミュニティそのものを反映したものであるべきであり、反響室を作らないように特定の企業に偏ったものではないと考えています。
        </li>
        <li>
            プロジェクトのリーダーは、すべての著者の推薦を検討し、新しい視点をもたらし、地域社会の中での未発表グループの声を増幅させる著者の選定に尽力しました。
        </li>
      </ul>
      
      <p>
        将来的には、このプロセスを繰り返していくことで、Web Almanacがより多様で包括的なプロジェクトとなり、さまざまなバックグラウンドを持つ投稿者が参加できるようにしたいと考えています。
```

Reply:
OK, I fix.